resources/aws/gateway.go

```diff
@@ -114,7 +114,7 @@ func (g *Gateway) Delete() error {
 		}
 		return nil
 	}
-	if err := backoff.Retry(detachOperation, backoff.NewExponentialBackOff()); err != nil {
+	if err := backoff.Retry(detachOperation, NewCustomExponentialBackoff()); err != nil {
```

Comment:
I would like to keep logging out of this PR, as I think finding a good solution is a more complicated change that will take more time.

I'd like to get this in fast, as it's currently reducing my ability to debug.

We can keep #174 for the logging discussion IMO.